docs/pktd/pktwallet.md

```diff
@@ -134,22 +130,20 @@ If you already have a wallet created on your computer called wallet.db you will
 1. Download [pktd-v1.5.1-macos.pkg](https://github.com/pkt-cash/pktd/releases/download/pktd-v1.5.1/pktd-v1.5.1-macos.pkg){target=_blank}
 2. Run the installation
 3. Open Terminal
-4. Enter the command: ```$ cd pktd```
-5. Enter the command: ```$ ~/pktd/bin/pktwallet --create --wallet xxxx```  
+4. Enter the command: ```$ ~/pktwallet --create --wallet xxxx```  
```

Comment:
Please remove `~/` in all cases, if it's been installed from the pkg then it will be in the $PATH, not in your $HOME. So you need `pktwallet` / `pktctl` rather than `~/pktwallet` ...